_data/library/scalalibs.yml

```diff
@@ -273,3 +277,4 @@
       dep: |
         addCompilerPlugin("org.scalamacros" % "paradise" % "2.1.0" cross CrossVersion.full)
         libraryDependencies += "com.thoughtworks.enableIf" %% "enableif" % "latest.release"
+
```

Comment:
Also tiny nitpick: you introduced a blank line that should be there. (there's supposed to be 1 EOL at the end of the file, because git prefers it that way, but not 2).